python/taichi/lang/matrix.py

```diff
@@ -1350,7 +1350,7 @@ def __getitem__(self, key):
 
     @python_scope
     def fill(self, val):
-        self.ndarray_matrix_fill(val)
+        self.ndarray_fill(val)
```

Comment:
What about removing this and renaming `ndarray_fill()` of the base `Ndarray` class to `fill()`?